TypeScript error TS1271: Decorator function return type is 'Record<..., unknown>' but is expected to be 'void' or 'any'.

I think there's a regression in the typedefs for version 1.3.1.

<img width="825" alt="TypeScript error TS1271: Decorator function return type is 'Record<..., unknown>' but is expected to be 'void' or 'any'." src="https://user-images.githubusercontent.com/1306747/161402789-5cf0e6cd-6b8f-40bd-acae-f08058779821.png">

https://github.com/github/catalyst/pull/190/files#diff-0609e410282ce718628c6b5977007b6a14a3edef8f178c61d749d134249ae52f

Changing the typedefs sorts it for me:

```diff
/**
 * Target is a decorator which - when assigned to a property field on the
 * class - will override that class field, turning it into a Getter which
 * returns a call to `findTarget(this, key)` where `key` is the name of the
 * property field. In other words, `@target foo` becomes a getter for
 * `findTarget(this, 'foo')`.
 */
- export declare function target<K extends string>(proto: Record<K, unknown>, key: K): Record<K, unknown>;
+ export declare function target<K extends string>(proto: Record<K, unknown>, key: K): void; 
/**
 * Targets is a decorator which - when assigned to a property field on the
 * class - will override that class field, turning it into a Getter which
 * returns a call to `findTargets(this, key)` where `key` is the name of the
 * property field. In other words, `@targets foo` becomes a getter for
 * `findTargets(this, 'foo')`.
 */
- export declare function targets<K extends string>(proto: Record<K, unknown>, key: K): Record<K, unknown>;
+ export declare function targets<K extends string>(proto: Record<K, unknown>, key: K): void;
//# sourceMappingURL=target.d.ts.map
```
*node_modules/@github/catalyst/lib/target.d.ts*

--- a/src/target.ts
+++ b/src/target.ts
@@ -7,8 +7,8 @@ import {findTarget, findTargets} from './findtarget.js'
  * property field. In other words, `@target foo` becomes a getter for
  * `findTarget(this, 'foo')`.
  */
-export function target<K extends string>(proto: Record<K, unknown>, key: K) {
-  return Object.defineProperty(proto, key, {
+export function target<K extends string>(proto: Record<K, unknown>, key: K): void {
+  Object.defineProperty(proto, key, {
     configurable: true,
     get() {
       return findTarget(this, key)
@@ -23,8 +23,8 @@ export function target<K extends string>(proto: Record<K, unknown>, key: K) {
  * property field. In other words, `@targets foo` becomes a getter for
  * `findTargets(this, 'foo')`.
  */
-export function targets<K extends string>(proto: Record<K, unknown>, key: K) {
-  return Object.defineProperty(proto, key, {
+export function targets<K extends string>(proto: Record<K, unknown>, key: K): void {
+  Object.defineProperty(proto, key, {
     configurable: true,
     get() {
       return findTargets(this, key)

